packages/ui/src/memberships/components/MemberListItem/components/MemberCreated.tsx

```diff
@@ -13,7 +13,9 @@ export const MemberCreated = React.memo(({ member }: MemberCreatedProps) => {
     <MemeberCreated>
       <MemberCreatedDate>{new Date(member.createdAt).toLocaleDateString('en-GB')}</MemberCreatedDate>
       {member.entry && member.entry.type !== 'genesis' && (
-        <MemberCreatedBlock>{member.entry.block.number} block</MemberCreatedBlock>
+        <MemberCreatedBlock>
+          {member.entry.block.number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')} block
+        </MemberCreatedBlock>
```

Comment:
Really nice fix. But I just realized that since the  `member.createdAt` is not correct but `member.entry.block.timestamp` is, this PR could also fix the wrong date (see https://github.com/Joystream/joystream/issues/4841), with something like:
```ts
const { entry, createdAt } = member
const block = entry.type === 'genesis' ? { number: 0, timestamp: createdAt } : entry.block
```
And then:
```tsx
      <MemberCreatedDate>{new Date(block.timestamp).toLocaleDateString('en-GB')}</MemberCreatedDate>
      <MemberCreatedBlock>{block.number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')} block</MemberCreatedBlock>
```